Add zone UI info (#24900)
home-assistant/home-assistant.github.io: source/_integrations/zone.markdown

Zones can be added and managed through the user interface at **{% my zones title="Settings -> Areas & Zones" %}**.

![Screenshot of the UI for adding or editing a zone](/images/integrations/zone/zone_edit_ui.png)

> In the user interface you can add a zone by specifying the GPS co-ordinates, or by dragging the icon. You can adjust the radius (except of the Home zone) by adjusting the size of the zone circle.

Reviewer: ```suggestion
You can add a zone in the user interface by specifying the GPS coordinates or dragging the icon on the map. You can adjust the zone radius (except for the Home zone) by changing the size of the zone circle.
```
